src/main/java/org/folio/dew/batch/acquisitions/edifact/services/ConfigurationService.java

```diff
@@ -24,7 +25,12 @@ public class ConfigurationService {
   private final ObjectMapper objectMapper;
 
   private ModelConfiguration getConfigById(String configId) {
-    return configurationClient.getConfigById(configId);
+    try {
+      return configurationClient.getConfigById(configId);
+    } catch (NotFoundException e) {
+      logger.error("getConfigById:: Couldn't fetch configuration entry by id: '{}'", configId, e);
+      throw new NotFoundException("Configuration entry not found for id: " + configId, e);
```

Comment:
can we just do throw e 
Or you need an another text in the excpetion message?

Reply:
Me and Bob encountered this issue and at first the exception message was a bit confusing like: `http://configurations/entries/uuid (NotFoundException)`. This will include same message but add some context why it was caused